lib/backend/syncersv1/updateprocessors/networkpolicyprocessor.go

```diff
@@ -58,11 +59,22 @@ func convertPolicyV2ToV1Spec(spec apiv2.PolicySpec) (interface{}, error) {
 		erules = append(erules, RuleAPIV2ToBackend(erule))
 	}
 
+	// If this policy is namespaced, then add a namespace selector.
+	selector := spec.Selector
+	if ns != "" {
+		nsSelector := fmt.Sprintf("%s == '%s'", apiv2.LabelNamespace, ns)
+		if selector == "" {
+			selector = nsSelector
+		} else {
+			selector += " && " + nsSelector
```

Comment:
I don't _think_ we generate any `||` clauses for these selectors.

If we did, we'd need `selector = fmt.Sprintf("%s && (%s)", nsSelector, selector)` or similar.

Reply:
To make sure the grouping is correct.

Reply:
yes i agree - and I think we DO need this for the user defined policy case